sn_cli/src/cli.rs

```diff
@@ -15,7 +15,7 @@ use crate::subcommands::SubCmd;
 #[command(author, version, about, long_about = None)]
 pub(crate) struct Opt {
     /// Nodes we dial at start to help us get connected to the network. Can be specified multiple times.
-    #[clap(long = "peer")]
+    #[clap(long = "peer", global = true)]
```

Comment:
Can we consider documenting this argument in the same way it is for the node?
https://github.com/maidsafe/safe_network/blob/0be2ef056215680b02ca8ec8be4388728bd0ce7c/sn_node/src/bin/safenode/main.rs#L70